### PR TITLE
Implement feature to dragg file item

### PR DESCRIPTION
**Pull request by kuzmavictor**
During this iteration was implemented:
- the moving file or folder to any folder;
- the new action for dragging file item.

### CODE REVIEW

**Comment by kuzmavictor** [comment]
@dkalinin fixed, PTAL.

**Comment by kuzmavictor** [comment]
@dkalinin fixed, PTAL.

File: app/service/store/action/index.js [hunk 1]
```diff
@@ -90,24 +106,32 @@ export const actions = {
...
+    store.commit('setOperationStatus', FILE_MOVING_MSG);
+    return store.requestServiceProvider.fileListService.moveFile(file, folderId)
+        .then((fileModel) => {
           store.dispatch('fetchFiles', store.state.files);
+        })
+        .catch((error) => {
+          store.commit('setMovingFileError', error);
+        })
+        .finally(() => {
+          store.commit('setOperationStatus', '');
```
Comment (dkalinin) [inline app/service/store/action/index.js]: This will clear "Loading files" message which is set from `fetchFiles`.

Comment (dkalinin) [reply app/service/store/action/index.js]: Please check other cases.

**Comment by kuzmavictor** [comment]
@dkalinin, PTAL.

**Pull request merged**